docs/pages/cloud/introduction.mdx

```diff
@@ -1,16 +1,33 @@
 ---
 title: Cloud
 description: Teleport as a service
-h1: Teleport cloud
+h1: Teleport Cloud
 videoBanner: 1jhKOtBinm4
 ---
 
-We run Teleport Cloud as hosted, managed Teleport as a service.
-Connect your nodes, web applications, kubernetes clusters and databases.
+# Teleport Cloud
+
+Teleport Cloud provides a SOC 2 certified Highly Available (HA) hosted Teleport Cluster service
+for your organization.  The same software used for Self-Hosted is managed by Teleport is deployed
```

Comment:
`is managed by Teleport is deployed` should probably be `is managed by Teleport and deployed`

Reply:
done